tensorflow/python/custom_zeta/custom_zeta

```diff
@@ -0,0 +1,14 @@
+# custom_zeta.py
+import tensorflow as tf
+import mpmath
+
+def custom_zeta(x, y):
+    # Convert TensorFlow tensors to mpmath complex numbers
+    s = mpmath.mpc(x.numpy(), y.numpy())
```

Comment:
Plus, this would only ever work in eager mode, where you can gain access to tensor values.  Most models are computed in graph mode (i.e. in a tf.function), where you wouldn't be able to do this.

Reply:
@cantonios Thank you so much for your input i have understood the requirements and i will come up with a better solution next time.

Reply:
okay thank you very much i understood